src/workingtitle-vcockpits-instruments-airliners/html_ui/Pages/VCockpit/Instruments/Airliners/Shared/WT/AltimeterIndicator.js

```diff
@@ -1639,20 +1630,10 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
                     this.pressurePresetLeftPart.textContent = leftPart;
                     this.pressurePresetCenterPart.textContent = ".";
                     this.pressurePresetRightPart.textContent = rightPart;
-
                     this.pressurePresetUnits.textContent = "IN";
                 }
             }
             else {
-                this.pressureBoxGroup.setAttribute("visibility", "hidden");
-                this.pressureSVGLeftPart.setAttribute("stroke", "black");
-                this.pressureSVGLeftPart.setAttribute("stroke-width", "5px");
-                this.pressureSVGCenterPart.setAttribute("stroke", "black");
-                this.pressureSVGCenterPart.setAttribute("stroke-width", "4px");
-                this.pressureSVGRightPart.setAttribute("stroke", "black");
-                this.pressureSVGRightPart.setAttribute("stroke-width", "5px");
-                this.pressureSVGUnits.setAttribute("stroke", "black");
-                this.pressureSVGUnits.setAttribute("stroke-width", "5px");
                 if (this.aircraft == Aircraft.A320_NEO) {

```

Comment:
same here

Reply:
```suggestion
                this.pressureBoxGroup.setAttribute("visibility", "hidden");
                if (this.aircraft == Aircraft.A320_NEO) {
```

Reply:
Sorry.

Reply:
No problem at all, could've been clearer myself.
It looks like we're good to go now!